Vagrantfile

```diff
@@ -2,11 +2,18 @@
 # vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
-  config.vm.box = 'ubuntu/trusty64'
+  # TODO: Switch back to the official box once it is released
```

Comment:
@antoviaque They seem to have an official box available for this now: https://vagrantcloud.com/ubuntu/boxes/xenial64. Might be worth trying to get it into this PR?